.github/workflows/check-pull-request.yaml

```diff
@@ -51,26 +63,26 @@ jobs:
           key: venv-${{ runner.os }}-${{ hashFiles('**/poetry.lock') }}
 
       #----------------------------------------------
-      # install dependencies if cache does not exist 
+      # install dependencies if cache does not exist
       #----------------------------------------------
       - name: Install dependencies
         if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
         run: poetry install --no-interaction --no-root
-        
+
+      #----------------------------------------------
+      #    install your root project, if required
       #----------------------------------------------
-      #    install your root project, if required 
-      #----------------------------------------------      
       - name: Install library
         run: poetry install --no-interaction
-          
+
 #       - name: Run image
 #         uses: abatilo/actions-poetry@v2.0.0
 #         # with:
 #         #   poetry-version: ${{ matrix.poetry-version }}
 #       - name: View poetry --help
 #         run: poetry --help
 
-      # Make test
-      - name: Make test
```

Comment:
Thanks @dalito for working on this for windows! :) - is this change for consistency, or does `make test` not work for windows?

Reply:
make is not available on windows.